manifests/s/Seedy-Threepio/Theseus2000/3B/Seedy-Threepio.Theseus2000.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Seedy-Threepio.Theseus2000
+PackageVersion: 3B
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.houseoffireseed.ml/programs/theseus/Theseus2000_Installer.exe
+  InstallerSha256: E358C7B7A16C56A6D5A73CCCBB2561D26C273A767CDE05CD39D837B9C16EFADB
+  AppsAndFeaturesEntries:
+  - DisplayName: Theseus2000 version 3

```

```suggestion
  - DisplayName: Theseus2000 version 3B
```
This must match Control Panel.